TACOS-rest/src/main/frontend/src/app/services/attribute/attribute.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable()
+export class AttributeService {
+
+  private readonly URL = "http://localhost:8080/TACOS-rest/api/v1/attributes";
+
+  constructor(protected httpClient: HttpClient) {
+  }
+
+  public createAttribute(attributeCreate: AttributeCreate) : Observable<AttributeCreate>  {
+    return this.httpClient.post<AttributeCreate>(this.URL, attributeCreate,{});
+  }
```

Comment:
The object u specify herec right after the post (post<AttributeCreate> ) Is the object returned by the create method. Now, we are regurning noly id, not the whole created object.